File: app/src/main/java/com/aquamorph/frcmanager/models/Award.java
package com.aquamorph.frcmanager.models;

/**
 * Contains information about awards given at an event
 *
 * @author Christian Colglazier
 * @version 12/27/2017
 */
public class Award {
	public String name;
//	public int award_type;
//	public String event_key;
	public RecipientList[] recipient_list;
//	public String year;

	public class RecipientList {
		public String team_key;
		public String awardee;
	}
}

File: app/src/main/java/com/aquamorph/frcmanager/models/Award.kt
package com.aquamorph.frcmanager.models

/**
 * Contains information about awards given at an event
 *
 * @author Christian Colglazier
 * @version 3/30/2018
 */
class Award {
    var name: String = ""
    var award_type: Int = 0
    var event_key: String = ""
    var recipient_list: Array<RecipientList> = emptyArray()
    var year: String = ""

    inner class RecipientList {
        var team_key: String = ""
        var awardee: String = ""
    }
}
